### PR TITLE
Navigation pages created

### PR DESCRIPTION
**Pull request by bryanmontalvan**
### Description
With this commit `Our Purpose, Community, Docs & Training, Op1st Community Cloud` each have their own respective pages along with a component for each. These pages come barebones with simple Mantine components (Buttons, Text, Title, etc)

**Next steps:**
The next steps after this PR is to come up with mock designs for how we would like each page to look and what content to have. Along with how should we link each Jupyter book

**Misc:**
Added `Link` component to `homepage/Navbar.js` so now the Operate First logo is clickable to the home page.

### Preview:
image

### CODE REVIEW

File: src/pages/community.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+// Component imports
+import { Nav } from "../components/homepage/Navbar";
+import { Footer } from "../components/homepage/Footer";
+import { CommunityContent } from "../components/nav-tabs/CommunityPage";
+
+const AboutPage = () => {
```
Comment (durandom) [inline src/pages/community.js:7]: would this be called `CommunityPage` ?

Comment (bryanmontalvan) [reply src/pages/community.js]: Yup, my mistake

**Pull request merged**